apps/screloc/main.cpp

```diff
@@ -281,17 +311,18 @@ class Reloc : public Client::Application {
 						relocated = true;
 					}
 					catch ( std::exception &e ) {
-						SEISCOMP_ERROR("  + processing failed - %s", e.what());
+						SEISCOMP_ERROR("Failed processing origin %s - %s",
+						               publicID.c_str(), e.what());
```

Comment:
I like the previous log more because the code doesn't stop after the error.

Reply:
The problem with the previous indentation is that it only works when running screloc with --debug. Otherwise, there is no top level line. This is why I suggest to remove the indentation.